docs/swagger/spec.yml

```diff
@@ -321,6 +476,89 @@ definitions:
         type: string
         description: URL safe version of name
         readOnly: true
+  LayerSource:
+    allOf:
+      - $ref: '#/definitions/BaseModel'
+      - type: object
+        properties:
+            name:
+              type: string
+              description: Display name for the layer
+            sourceURI:
```

Comment:
Oh okay, based off the v1 API I was working under the assumption that a Layer would be associated with only one image (but that image could have multiple bands). This is a pretty easy change to make I think.